xml/System.Security.Cryptography/DSACng.xml

```diff
@@ -596,11 +592,12 @@
         <Parameter Name="signature" Type="System.ReadOnlySpan&lt;System.Byte&gt;" Index="1" FrameworkAlternate="netcore-3.0" />
       </Parameters>
       <Docs>
-        <param name="hash">To be added.</param>
-        <param name="signature">To be added.</param>
-        <summary>To be added.</summary>
-        <returns>To be added.</returns>
+        <param name="hash">A read-only span of bytes containing the signed data.</param>
+        <param name="signature">A read-only span of bytes containing the signature to be verified.</param>
+        <summary>Verifies if the specified digital signature matches the specified data.</summary>
+        <returns><see langword="true" /> if <paramref name="rgbSignature" /> matches the signature computed using the specified data; otherwise, <see langword="false" />.</returns>
```

Comment:
```suggestion
        <returns><see langword="true" /> if <paramref name="signature" /> matches the signature computed using the specified data; otherwise, <see langword="false" />.</returns>
```